lib/grpc/GrpcService.ts

```diff
@@ -991,6 +992,35 @@ class GrpcService {
     }
   };
 
+  /*
+   * See [[Service.subscribeAlerts]]
+   */
+  public subscribeAlerts: grpc.handleServerStreamingCall<xudrpc.SubscribeAlertsRequest, xudrpc.Alert> = (call) => {
+    if (!this.isReady(this.service, call)) {
+      return;
+    }
+
+    const cancelled$ = getCancelled$(call);
+    this.service.subscribeAlerts((serviceAlert: Alert) => {
+      const alert = new xudrpc.Alert();
+      alert.setType(serviceAlert.type as number);
+      alert.setMessage(serviceAlert.message);
+      alert.setDate(serviceAlert.date);
+      if (serviceAlert.type === AlertType.LowTradingBalance) {
+        const balanceServiceAlert = serviceAlert as Alert;
+        const balanceAlert = new xudrpc.BalanceAlert();
+        balanceAlert.setBound(balanceServiceAlert.bound);
```

Comment:
Bound is not in the xudrpc.proto definition currently. Regenerating the protos results in compile errors currently. Do we want this field?

Reply:
fixed